sdk/tests/iot/hub/main.c

```diff
@@ -15,12 +15,12 @@ int main()
 {
   int result = 0;
 
-  result += test_iot_hub_c2d();
   result += test_iot_hub_client();
-  result += test_iot_sas_token();
-  result += test_iot_hub_telemetry();
-  result += test_az_iot_hub_client_twin();
-  result += test_iot_hub_methods();
+  result += test_iot_hub_client_c2d();
```

Comment:
Why not use test_az_iot_hub_client_x for these?
I notices that naming structure is used in these files, but it is for the static functions.  This non-static is the exception.

Reply:
Yea my original logic was api's shouldn't be prefix'd with `az_` unless they were public but these are prefix'd with `test_`. I blame my lack of coffee this morning. Will change

Reply:
are you thinking the static functions lose the az, and the public one gains it?  Or something else?